php-cs-fixer

```diff
@@ -35,6 +35,7 @@ set_error_handler(static function (int $severity, string $message, string $file,
         exit(1);
     }
 
+    // @phpstan-ignore smaller.alwaysFalse
```

Comment:
is it possible to add a comment why we put this ignore, in-code ? so john doe won't wonder